spec/unit/packagers/rpm_spec.rb

```diff
@@ -158,6 +172,14 @@ module Omnibus
         expect(contents).to include("Obsoletes: old-project")
       end
 
+      it 'supports the epoch tag' do
```

Comment:
Can we also add a test for the case when an epoch isn't exlicitly set to ensure that the spec doesn't include the Epoch: line?